pkg/sql/schemachanger/scplan/plan_test.go

```diff
@@ -191,3 +193,84 @@ func marshalOps(t *testing.T, plan *scplan.Plan) string {
 	}
 	return stages.String()
 }
+
+// TestPlanGraphSort sanity checks sorting of the graph.
```

Comment:
it seems like this test can go in `scgraph`. I wonder if we can rework it to be easier to read and extend.